cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -378,9 +383,7 @@ def export_status_handler(request, course_key_string):
                 'response-content-type': 'application/x-tgz'
             })
         else:
-            # local file, serve from the authorization wrapper view
-            output_url = reverse_course_url('export_output_handler', course_key)
-
+            output_url = artifact.file.storage.url(artifact.file.name)
```

Comment:
This appears to be new logic, do you know if this is covered in any of the unit tests? If not, would you please add a test for it.

Reply:
There don't appear to be any tests for this function at all :-(

Reply:
Sorry about that, I found the tests, they were just mixed up with the export_status tests. I've added some tests now.